src/mmw/js/src/data_catalog/views.js

```diff
@@ -527,9 +540,29 @@ var ResultDetailsCuahsiView = ResultDetailsBaseView.extend({
     },
 
     onRender: function() {
-        this.valuesRegion.show(new CuahsiTableView({
-            collection: this.model.get('variables')
-        }));
+        this.showValuesRegion();
+    },
+
+    showValuesRegion: function() {
+        var mode = this.model.get('mode'),
+            variables = this.model.get('variables'),
+            view = mode === 'table' ?
+                   new CuahsiTableView({ collection: variables }) :
```

Comment:
For some of our other toggles, we render all views [`onShow` (or wherever)](https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/js/src/modeling/views.js#L959) and then do the switching via `display: none;` styling on the non-active view (bootstrap might take care of this?). 

I find the way you implemented it here a lot easier to follow, but I think the other way reduces the amount of rendering needed.